test/Chainweb/Test/Pact/PactInProcApi.hs

```diff
@@ -445,10 +489,19 @@ _pact420UpgradeTest bdb mpRefIO pact = do
       where
         expressions =
           [
-            "(distinct [1 1 2 2 3 3])"
-          , "(concat [\"this\" \"is\" \"a\" \"test\"])"
-          , "(str-to-list \"test\")"
-          , "(enumerate 1 10)"
+            "(let* ((qry (lambda (k o) (<  k \"c\"))) (consume (lambda (k o) k))) (fold-db free.fdb.fdb-tbl (qry) (consume)))"
```

Comment:
you can inline the `lambda` in consume, we added this feature,m so that expression at the end can be
```lisp
(fold-db free.fdb.fdb-tbl (qry) (lambda (k o) k))
```
and it should give you the same thing. The nice part about it is that it does add a bit of an extra test that the inline lambdas are working just fine (though we do test it in pact as well).